### ISSUE TITLE
Mark apple touch icons as precomposed

### ISSUE THREAD

**Issue by san650** [san650/ember-web-app#29]
Register images as

```html
<link rel="apple-touch-icon-precomposed" href="apple-touch-iphone.png" >
```

instead of

```html
<link rel="apple-touch-icon" href="apple-touch-iphone.png" >
```

I understand that the `precomposed` flag indicates apple to use the images as-is (without adding gloss etc.).

See http://stackoverflow.com/questions/5110776/apple-touch-icon-for-websites

**Comment by hidnasio** [comment]
I made the pull request https://github.com/san650/ember-web-app/pull/31 to add this feature.

**Comment by san650** [comment]
Fixed by #31
